guide/taxonomy-ingestion.md

```diff
@@ -76,65 +67,24 @@ with a body like:
     // ensure values in Zentail with differing units are converted properly
     // before including them on the listing.
     "unit": "string",
+    "can_be_pivot": true,
     "level": "<one of LEVEL_LISTING, LEVEL_VARIANT>",
+    "usage": "<one of USAGE_AVAILABLE, USAGE_RECOMMENDED, USAGE_REQUIRED>",
     "classification": "<one of CLASSIFICATION_PRODUCT, CLASSIFICATION_INVENTORY, CLASSIFICATION_PRICING, CLASSIFICATION_LOGISTICS, CLASSIFICATION_IDENTIFIER>",
   }
 }
 ```
 
-> [!NOTE]
-> Attribute Specs are always associated with a category,
-> if a certain Attribute is available in multiple categories,
-> you should invoke the `CreateAttributeSpec` action for each category ID
-
-## Product Types
+### Listing Level
 
-At this point, we have established the bear minimum for generating a listing
-from Zentail. The next 2 sections outline tools you can use to provide greater
-visibility into what makes a great listing on your channel. `ProductType`s and 
-`QualityAttributeSpec` drive Listing Quality workflows in Zentail that help sellers
-understand requirements and recommendations for each Listing.
+The `level` property can be used to enforce that certain values must share a common value
```

Comment:
this isn't totally clear to me, it should say to use `LEVEL_LISTING` when you want to ensure that an attribute needs to have the same value for all variants.  i think the next callout saying it's fine to just put LEVEL_VARIANT for everything looks good